### PR TITLE
feat(README): add more details on purpose, process and structure

### PR DESCRIPTION
**Pull request by ahmadnassri**
- remove issues from this repo to ensure process is followed
- update pull request template to reflect process details
- update README with details

**DO NOT MERGE UNTILL ALL EXISTING ISSUES ARE CLOSED**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
-> This is a collaborative effort to document our digital standards, tooling and process, you are welcome to contribute and suggest changes, please follow the [contribution guidelines](.github/CONTRIBUTING.md)
-
 ## Why
 
-For new and existing team members & partners, a single place where all the technical knowledge and platform specifications can be documented, tracked and debated.
+> _Why does this exist?_
+
+This wiki acts as a source of truth for all the technical knowledge, software specifications, standards and architecture evolution.
+
+This is a collaborative effort to **documented**, **track**, and **discuss** digital software standards, tooling and practices, you are welcome to contribute and suggest changes, please follow the [contribution guidelines](.github/CONTRIBUTING.md) and keep in mind: _this repository is [**Open Source**](#open-source)_.
```
Comment (ruxandrafed) [inline README.md]: Typo "documented"

**Comment by ruxandrafed** [comment]
@ahmadnassri Pls see the typo I mentioned and fix the liner 😄

File: README.md (hunk 1 above)
Comment (ahmadnassri) [reply README.md]: :+1:

**Comment by ahmadnassri** [comment]
I choose to keep the style and override the linter for this one rule as in this case, the style in putting the emphasis on the text is important.

**Pull request merged**